content/docs/command-reference/import-url.md

```diff
@@ -125,12 +125,17 @@ source.
   default file name: `<file>.dvc`, where `<file>` is the desired file name of
   the imported data (`out`).
 
-- `--no-exec` - create `.dvc` file without actually downloading `url`. E.g. if
-  the file or directory already exists, this can be used to skip the download.
-  The data hash is not calculated when this option is used, only the import
-  metadata is saved to the `.dvc` file. `dvc commit <out>.dvc` can be used if
-  the data hashes are needed in the `.dvc` file, and to save existing data to
-  the cache.
+- `--no-exec` - create the import `.dvc` file without actually downloading
+  `url`. No connection to the data source is made, so only the import operation
```

Comment:
`No connection to the data source is made` I don't quite understand this :(

We have 3 paragraphs for a quite simple option - we should simplify it significantly. it's very repetitive now.

Reply:
same in other case, other commands in this PR

Reply:
> No connection to the data source is made

It was meant originally to explain why we can't even add hash values with this option. But OK, I just removed that now.

I also removed the 2nd p about scenarios but left the note about not knowing whether the data source is valid until attempting the download.